src/Action_RandomizeIons.cpp

```diff
@@ -1,7 +1,11 @@
 #include <cmath> // sqrt
-#include <cstdlib> // random, srandom
+#include <cstdlib> // srandom, random
```

Comment:
+1 for reworking to use `Random_Number`.  For now it works so we can do this in a separate PR if you want to wait on it.  If you wanted to make this change quickly, then let's do it in this PR so we avoid merge conflict issues.